src/main/java/cz/smarteon/loxone/message/TextEvent.java

```diff
@@ -1,15 +1,27 @@
 package cz.smarteon.loxone.message;
 
 import cz.smarteon.loxone.LoxoneUuid;
+import lombok.Getter;
+import lombok.ToString;
 import org.jetbrains.annotations.NotNull;
 
 import static java.util.Objects.requireNonNull;
 
 /**
  * Loxone event carrying text value.
  */
+@Getter
+@ToString(callSuper = true)
 public class TextEvent extends LoxoneEvent {
+
+    /**
```

Comment:
does lombok somehow move this to getters? Otherwise it makes no sense to document it - prvate members are usually not visible in api doc

Reply:
Documentation on fields is copied to the getters and setters at compile time.